master/buildbot/www/rest.py

```diff
@@ -289,7 +298,7 @@ def checkFields(fields, negOk=False):
                                       offset=offset, order=order, filters=filters)
 
         # for singular endpoints, only allow fields
-        if not endpoint.isCollection:
+        if not endpoint.isCollection and not properties:
```

Comment:
so that is not needed. Indeed, filters, are to filter a collection. Field is to add optional fields.